src/AbstractAlgebra.jl

```diff
@@ -211,7 +211,7 @@ import .Generic: add!, addeq!, addmul!, add_column, add_column!, add_row, add_ro
                  newton_to_monomial!, ngens, normalise, nrows, nvars, O, one,
                  order, ordering, parent_type, parity, partitionseq, Perm, perm,
                  permtype, @perm_str, polcoeff, pol_length, powmod,
-                 pow_multinomial, popov, popov_with_transform, powers,
```

Comment:
I don't understand why powers has to be removed here

Reply:
It's because before methods were defined both in `AbstractAlgebra` and `Generic` modules, in this PR there is only one method, living in the "NCRings.jl" file, i.e. not methods are defined anymore in `Generic`, so the function can belong to the `AbstractAlgebra` module.